index.html

```diff
@@ -65,147 +66,158 @@ <h1 class="intro">I'm Ernest Glad to see you!</h1>
+                        <li class="canopy">CANOPY</li>
+                        <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                        <li class="back-end-dev">BackEndDev</li>
+                        <li><img src="./images/Counter.png" alt="counter"/></li>
+                        <li class="year">2022</li>
+                    </ul>
+                    
+                    <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+                </div>
+        
+                <ul class="programing-languages">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
                 </ul>
-                
-                <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+        
+                <button type="button">See Project</button>
             </div>
-    
-            <ul class="programing-languages">
-                <li><a href="#">html</a></li>
-                <li><a href="#">css</a></li>
-                <li><a href="#">javaScript</a></li>
-            </ul>
-    
-            <button type="button">See Project</button>
         </div>
 
         <!--Professional Section-->
         <div class="Professional padding">
             <div class="frontpage-icon">
                 <img src="./images/Professional.png" alt="Portfolio">
             </div>
-            <div class="tonic">
-                <h2>Multi-Post Stories</h2>
-                <ul class="tonic-section-header">
-                    <li class="canopy">CANOPY</li>
-                    <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
-                    <li class="back-end-dev">BackEndDev</li>
-                    <li><img src="./images/Counter.png" alt="counter"/></li>
-                    <li class="year">2022</li>
+            <div class="tonic-wrapper">
+                <div class="tonic">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="tonic-section-header">
+                        <li class="canopy">CANOPY</li>
+                        <li><img src="./images/Counter.png" alt="counter" class="counter"/></li>
+                        <li class="back-end-dev">BackEndDev</li>
+                        <li><img src="./images/Counter.png" alt="counter"/></li>
+                        <li class="year">2022</li>
+                    </ul>
+                    
+                    <p class="tonic-details">A daily selection of privately personalized reads: no accounts or sign-ups required</p>
+                </div>
+        
+                <ul class="programing-languages">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
```

Comment:
- [ ] Kindly make the `li` tag similar to the Figma design as the background color is not matching.
### your design
![hiiii](https://user-images.githubusercontent.com/81727988/171616478-27286d33-82cd-43be-aa33-772baaad5d0c.png)
# Figma design
![figmaaa](https://user-images.githubusercontent.com/81727988/171616448-4872004c-7e7e-428f-b8a9-20357f415119.png)